### DISCUSSION TITLE
Mohr coulomb plasticity

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2212] [Q&A]
Hello Radioss expert,

I am wandering is there any material laws in OpenRadioss can include Mohr Coulomb Plasticity? Many Thanks

Regards
Lee

**Comment by MarianBulla** [comment]
Hello li2qn, 
and Happy New Year 2024 :-) 

Are you going to model geological materials ?
If "yes", than you may have a look at 
- LAW10
- LAW21
- LAW24
- LAW81
- LAW102

They are mostly based on the Drucker-Prager approach, which can sometimes also be used for modelling of Mohr Coulomb materials.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Hi Marian, Thanks for printing out these. It's helped. 

Best
Regards
